indico/modules/rb/controllers/backend/bookings.py

```diff
@@ -345,11 +358,27 @@ def _process(self, args):
             additional_booking_attrs['new_booking_id'] = new_booking.id
 
         db.session.flush()
+        self.after_update(args, reservation=self.booking, new_split_reservation=new_booking)
         today = date.today()
         calendar = get_room_calendar(args['start_dt'] or today, args['end_dt'] or today, [args['room_id']])
         return jsonify(booking=(serialize_booking_details(self.booking) | additional_booking_attrs),
                        room_calendar=list(serialize_availability(calendar).values()))
 
+    @staticmethod
+    @make_interceptable
+    def after_update(
+            args: dict[str, typing.Any],
+            reservation: 'Reservation',
+            new_split_reservation: typing.Optional['Reservation'] = None,
+            **kwargs,
```

Comment:
Same here

```suggestion
            args: dict[str, typing.Any],
            reservation: Reservation,
            new_split_reservation: typing.Optional[Reservation] = None,
            **kwargs,
```

Reply:
Updated in https://github.com/indico/indico/pull/6126/commits/07b7a5b4c8cfa60b78a1729483d63164aae66801, thanks!